iFixit/react-commerce
# fix(strapi): Update dependencies and Dockerfile

fixes #268 

## Changelog

- Refactor Dockerfile
- Use Yarn to install dependencies
- Update Strapi to v4.1.8

## QA

1. Open https://fix-broken-admin-dashboard.govinor.com/admin
2. Verify that you are able to login and the Strapi dashboard loads correctly
